<?xml version="1.0" encoding="UTF-8"?> 
<launch>
  <node name="tf_to_path" pkg="tf_to_trajectory" type="tf_to_path.py"/>
  <param name="/parent_frame_id" value="/world"/>
  <param name="/child_frame_id" value="/imu"/>
</launch>
